### ISSUE TITLE
Avatar Hover : Problem with themes

### ISSUE THREAD

**Issue by BigBoss320** [Inve1951/BetterDiscordStuff#95]
Hello
I have a problem with the plug-in when I put a theme. The picture doesn't work (ScreenShots)

Without theme :
![image](https://user-images.githubusercontent.com/44122706/77479068-67a30580-6e1f-11ea-964b-18e02b6de29f.png)
 With theme :
![image](https://user-images.githubusercontent.com/44122706/77479164-8ef9d280-6e1f-11ea-9613-0225e3b45cbd.png)

**Comment by Inve1951** [comment]
Hi, sorry for the late response.

The hover card is designed to make it easy for themes to detect and customize it. Generally speaking it's the theme author's responsibility to maintain plugin compatibility since it's impossible to guess what all the themes out there change on specific elements and also to ensure selector specificity.

TLDR; Ask your theme's author to fix it.
